File: Ad-Hoc/__3000__/3129/3129.php
<?php

$stickers = [];
$repeated_stickers = 0;

$n = (int) fgets(STDIN);
for($i = 0; $i < $n; $i++){
    $sticker = (int) fgets(STDIN);
    $len = count($stickers);
    $stickers[$sticker] = true;
    if(count($stickers) == $len) $repeated_stickers++;
}

echo count($stickers) . "\n";
echo "$repeated_stickers\n";

File: Ad-Hoc/__3000__/3129/3129.cpp
#include <iostream>
#include <set>
using namespace std;

int main(){

    set<int> stickers;
    int n, curr, repeated_stickers = 0;
    cin >> n;

    for(int i = 0; i < n; i++){
        cin >> curr;
        int len = stickers.size();
        stickers.insert(curr);

        repeated_stickers += len == stickers.size() ? 1 : 0;
    }

    cout << stickers.size() << endl;
    cout << repeated_stickers << endl;

    return 0;
}